go.mod

```diff
@@ -46,7 +46,6 @@ require (
 	github.com/google/flatbuffers v2.0.6+incompatible
 	github.com/google/go-cmp v0.6.0
 	github.com/google/uuid v1.6.0
-	github.com/gotesttools/gotestfmt/v2 v2.4.1
```

Comment:
I assume this is after running `go mod tidy`

Reply:
Yep, one test tool was an indirect dependency of gostream's 'tool.go` unused in rdk.